src/ckb/channel.rs

```diff
@@ -2441,6 +2433,18 @@ impl ChannelActorState {
         KeyAggContext::new(keys).expect("Valid pubkeys")
     }
 
+    pub fn get_local_funding_script(&self) -> Script {
```

Comment:
Can we rename it to `get_default_local_funding_script`? To some unfamiliar eyes, the name `get_local_funding_script` imply that we will get the intended funding script.